metadata.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml b/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml
+index 513f18c..89ff3a8 100644
+--- a/butt/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml
++++ b/butt/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml
+@@ -1,23 +1,53 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop-application">
+   <id>de.danielnoethen.butt</id>
+-  
++
+   <name>butt</name>
+   <summary>Icecast and Shoutcast streaming client</summary>
+-  
++  <developer_name>Daniel Nöthen</developer_name>
++  <developer id="de.danielnoethen">
++    <name>Daniel Nöthen</name>
++  </developer>
++
+   <metadata_license>MIT</metadata_license>
+   <project_license>GPL-2.0-or-later</project_license>
+-  
++
+   <description>
++    <p>
++      NOTE: This wrapper is not verified by, affiliated with, or supported by the developer. Therefore, and so to offer the best support and experience for the Flatpak wrapper please report ALL issues at the Flathub issue tracker (linked by the issue button/link on this page). This allows with issues to be dealt with correctly. For more info press the issues button/link on this page. 
```

Comment:
you build from source, and the license is Libre. this disclaimer is not necessary.